backend/src/controllers/orders-controller.ts

```diff
@@ -66,6 +73,8 @@ export const cancelOrder = async (req: Request, res: Response) => {
             res.status(404).json({ error: 'Order not found' });
         }
         await Order.updateOne({ _id: new ObjectId(orderId) }, { $set: { status: 'denied' } });
+        await sendEmail(order?.customerEmail, "Customer: Order Canceled", "Your order ${orderId} has been canceled.", `<p>Your order ${orderId} has been canceled.</p>`)
+        await sendEmail(req.body.email, "Seller: Order Canceled", "Order ${orderId} has been canceled by ${order?.customerEmail}.", `<p>Order ${orderId} has been canceled.</p>`)
```

Comment:
(Repeat) Seller email is not in `req.body.email`. Pull the user object from `itemOwnerId` of the order, then get their email from that.